main.go

```diff
@@ -79,14 +79,12 @@ func handlerHelp(ctx context.Context, telegramBot *bot.Bot, update *models.Updat
 }
 
 func handlerSendPhoto(ctx context.Context, tgBot *bot.Bot, update *models.Update) {
-	// TODO: Set actual logic whenever the message is a reply lmao
-	/*
-		    if update.Message.ReplyToMessage != nil {
-			handlerSendPhoto(ctx, bot, update)
-		    } else {
-
-		    }
-	*/
+	
+	if isReply(update) {
```

Comment:
handlerSendPhoto is only called whenever a "/cat" command is received per line 25:
```go
bot.WithMessageTextHandler("/cat", bot.MatchTypeExact, handlerSendPhoto),
```
running this command in here would be redundant; instead we should have a general Handler for every message that is a reply.